### PR TITLE
Adding SSL troubleshooting

### PR DESCRIPTION
**Pull request by it33**

### CODE REVIEW

File: source/deployment/push.rst [hunk 1]
```diff
@@ -215,3 +215,29 @@ These procedures summarize all potential bottlenecks in a system for mobile app
...
+If the login doesn't work, please report an issue to https://github.com/mattermost/platform/issues
+
+If the login does work: 
+
+1. Check that the SSL URL is properly installed by entering it in a certificate checker, such as: https://cryptoreport.websecurity.symantec.com/checker/
+2. Correct any issues with your certificate 
+3. Try connecting to the HTTPS URL of your server using the mobile app
+4. If you're still having issues please `open a new topic in the troubleshooting forum <https://forum.mattermost.org/c/general/trouble-shoot>`_ with steps to reproduce your issue. If you're an Enterprise Edition subscriber, you can also email subscribers@mattermost.com for support. 
+
+Note: Mobile apps do not support self-signed certificates. To use free certificates signed by a Certificate Authority, visit https://letsencrypt.org/
```
Comment (coreyhulen) [inline source/deployment/push.rst]: We also do not support client side certs.

**Comment by coreyhulen** [comment]
otherwise +1

**Comment by coreyhulen** [comment]
this also feels like it's in the wrong place.

**Comment by it33** [comment]
@esethna I'm going to commit this and leave the PM Review label on. I agree with Corey this isn't the best place for this troubleshooting guide and will defer to you on where to put it finally, this is just needed to answer some customer issues right now.

**Pull request merged**